mdsuite/calculators/trajectory_calculator.py

```diff
@@ -169,12 +171,12 @@ def _string_to_function(argument):
             switcher = {**switcher_unwrapping, **switcher_transformations}
 
             choice = switcher.get(
-                argument, lambda: "Data not in database and can not be generated."
+                argument, lambda: "Data not in database and cannot be generated."
             )
             return choice
```

Comment:
Not related to the PR, because this is only formatting, but this confuses me. I think this should either raise and Error or use logging instead of what it is doing here.

Reply:
Yes it can be changed. It was the default Python example of implementing a switch case and is very old in the code. There were more of them but now I think this may be the last one.